content/en/continuous_integration/pipelines/jenkins.md

```diff
@@ -142,7 +143,98 @@ If your Jenkins instance uses the Jenkins [`configuration-as-code`][1] plugin:
             targetCredentialsApiKey: "<your-api-key>"
             # (Optional) Configure your CI Instance name
             ciInstanceName: "jenkins"
-    ```
+    {{< /code-block >}}
+
+    {{% /site-region %}}
+    {{% site-region region="eu" %}}
+
+    {{< code-block lang="yaml" disable_copy="true" collapsible="true" >}}
+    unclassified:
+        datadogGlobalConfiguration:
+            # Select the `Agentless` mode (HTTP).
+            reportWith: "HTTP"
+            # Update the endpoints if reporting to Datadog sites other that `datadoghq.com`
```

Comment:
I guess these `# Update the endpoints if reporting to Datadog sites other that datadoghq.com` comments are less relevant now that the URLs are updated automatically.
Perhaps we could remove them, or reword to say something like `# Endpoint URLs correspond to your Datadog site`